pypgx/api/data/diplotype-table.csv

```diff
@@ -4356,4 +4356,1685 @@ UGT1A1,*28/*80+*37,Poor Metabolizer
 UGT1A1,*36/*80+*37,Intermediate Metabolizer
 UGT1A1,*37/*80+*37,Poor Metabolizer
 UGT1A1,*80/*80+*37,Indeterminate
-UGT1A1,*80+*37/*80+*37,Poor Metabolizer
\ No newline at end of file
+UGT1A1,*80+*37/*80+*37,Poor Metabolizer

```

Comment:
1. I see that you added CFTR diplotypes at the end of the file. Because I try to keep the alphabetical order of the genes, please move them after CACNA1S.
2. Please change "Increased open channel probability" and "Significantly enhanced channel open probability" to "Favorable Response".
3. Please change "No change in open channel probability" to "Unfavorable Response".
4. I'm not sure why but you have a lot of duplicate records such as "Reference/F508del". Please make sure you are generating a unique set of records. Here's the code I used for RYR1:

```
import pypgx
import itertools
alleles = pypgx.list_alleles('RYR1')
diplotypes = list(itertools.combinations(alleles, 2))
diplotypes = [x[0] + '/' + x[1] for x in diplotypes] # heterozygotes
diplotypes += [x + '/' + x for x in alleles] # homozygotes
```
There should be 861 unique CFTR diplotypes.

Reply:
Yes, I forgot to drop duplicated entries from `[p for p in itertools.product(alleles, repeat=2)]`. I will update this file as soon as we agree on the phenotype.